docs/DataProviderList.md

```diff
@@ -45,6 +45,8 @@ If you can't find a Data Provider for your backend below, no worries! [Writing a
 * **[PostgREST](https://postgrest.org/)**: [raphiniert-com/ra-data-postgrest](https://github.com/raphiniert-com/ra-data-postgrest)
 * **[Prisma](https://github.com/weakky/ra-data-prisma)**: [weakky/ra-data-prisma](https://github.com/weakky/ra-data-prisma)
 * **[Prisma Version 2](https://www.prisma.io/)**: [panter/ra-data-prisma](https://github.com/panter/ra-data-prisma)
+* **[Prisma](https://www.npmjs.com/package/ra-data-simple-prisma)**: [codeledge/ra-prisma](https://github.com/codeledge/ra-prisma)
+
```

Comment:
should not have impact on the markdown output but perhaps better not having a new line for consistency

Reply:
ok